plugins/operator.yaml

```
@@ -0,0 +1,46 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: operator
+spec:
+  version: v0.0.3
+  homepage: https://github.com/operator-framework/kubectl-operator
+  shortDescription: Manage Kubernetes Operators with Operator Lifecycle Manager
```

```suggestion
  shortDescription: Manage operators with Operator Lifecycle Manager
```
Everything above 50 chars is usually truncated.